text_collector_examples/multipathd_info

```diff
@@ -0,0 +1,9 @@
+#/bin/bash
```

Comment:
Shouldn't this be `#!/bin/bash`?

Reply:
This doesn't need to be bash. There is nothing in this script that uses bash-specific features, so it should simply be /bin/sh.

Reply:
Shouldn't it be `#!/bin/sh`? ;-)

I don't see any bash-specific code in the script.